spec/models/planning_application_spec.rb

```diff
@@ -2164,19 +2164,21 @@
       end
     end
 
-    context "when STAGING_ENABLED env variable is set to true" do
+    context "when environment is production" do
       before do
-        allow(ENV).to receive(:fetch).with("STAGING_ENABLED", "false").and_return("true")
+        allow(ENV).to receive(:fetch).and_call_original
+        allow(ENV).to receive(:fetch).with("BOPS_ENVIRONMENT", "development").and_return("production")
```

Comment:
Maybe this is nitpicking, but I think perhaps this stub should apply regardless of what the second parameter to `fetch` is:

```suggestion
        allow(ENV).to receive(:fetch).with("BOPS_ENVIRONMENT", any_args).and_return("production")
```

But perhaps it doesn't matter, if there's only one place that we're looking up the environment variable.

Reply:
Yes that's true. I didn't want to change the logic, just refactor it and be more specific where the previous env vars were set so we don't have random failures 😄